### PR TITLE
Update project-features.adoc

### LINKED ISSUE #1594: Wrong link to the advanced docs
https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/documentation-overview.html#advanced-topics points to `advanced.html.html`

https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/project-features.html#features-whats-next doesn't point to `advanced.html`

the documentation code can be found under `docs/src/main/asciidoc`

### PR DESCRIPTION
**Pull request by Utkarsh299-tech**
Fixes #1594
Updated the link to advanced docs.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Utkarsh299-tech Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=1595)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=1595) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Utkarsh299-tech Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-contract&pullRequestId=1595)!

File: docs/src/main/asciidoc/project-features.adoc [hunk 1]
```diff
@@ -37,4 +37,4 @@ If you want to learn more about any of the classes discussed in this section, yo
 
 If you are comfortable with {project-full-name}'s core features, you can continue on and read
 about
-<<advanced.html, {project-full-name}'s advanced features>>.
+<<documentation-overview.adoc#advanced-topics, {project-full-name}'s advanced features>>.
```
Comment (marcingrzejszczak) [inline docs/src/main/asciidoc/project-features.adoc:40]: I don't think that this is correct. What should change is that we link to the `advanced.html` file. @Buzzardo am I correct that you have to use the `link:` approach?

Also that's only `project-features` file. There was another reference in the `documentation-overvivew.adoc` that is wrong and needs to be fixed.

**Review by marcingrzejszczak — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/project-features.adoc (hunk 1 above)
Comment (Utkarsh299-tech) [reply docs/src/main/asciidoc/project-features.adoc:40]: Hey, @marcingrzejszczak! Do you mean I should link to the advanced.html file?

Comment (marcingrzejszczak) [reply docs/src/main/asciidoc/project-features.adoc:40]: Yes - https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/advanced.html

**Comment by marcingrzejszczak** [comment]
BTW @Utkarsh299-tech if you want to verify that things work fine, just run

```
$ ./mvnw clean install -Pdocs -pl docs
```

and then you'll generate the documentation. It will be available under `docs/target/generated-docs/reference/`

**Comment by Utkarsh299-tech** [comment]
> BTW @Utkarsh299-tech if you want to verify that things work fine, just run
> 
> ```
> $ ./mvnw clean install -Pdocs -pl docs
> ```
> 
> and then you'll generate the documentation. It will be available under `docs/target/generated-docs/reference/`

Okay, thanks!

File: docs/src/main/asciidoc/project-features.adoc (hunk 1 above)
Comment (Utkarsh299-tech) [reply docs/src/main/asciidoc/project-features.adoc]: > I don't think that this is correct. What should change is that we link to the `advanced.html` file. @Buzzardo am I correct that you have to use the `link:` approach?
> 
> Also that's only `project-features` file. There was another reference in the `documentation-overview.adoc` that is wrong and needs to be fixed.

Can you please be more specific about the reference to fix in `documentation-overview.adoc` ?

Comment (marcingrzejszczak) [reply docs/src/main/asciidoc/project-features.adoc]: I've described it in the issue description https://github.com/spring-cloud/spring-cloud-contract/issues/1594

> https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/documentation-overview.html#advanced-topics points to advanced.html.html

Comment (Utkarsh299-tech) [reply docs/src/main/asciidoc/project-features.adoc]: > I've described it in the issue description #1594
> 
> > https://docs.spring.io/spring-cloud-contract/docs/3.0.0/reference/html/documentation-overview.html#advanced-topics points to advanced.html.html

Yes, thanks! I got it and I've tried to fix it. Please review the PR.

**Comment by marcingrzejszczak** [comment]
Congrats @Utkarsh299-tech on your first contribution! :sparkles:

**Comment by Utkarsh299-tech** [comment]
> Congrats @Utkarsh299-tech on your first contribution! ✨

Oh thank you for merging the PR. I'm glad to contribute to this project. 😀

**Pull request merged**